src/stories/50_User_Consent/Story_User_Consent.html

```diff
@@ -0,0 +1,167 @@
+
+  <!-- ## Setup -->
+  <!-- Import the `amp-story` extension for creating an AMP Story. -->
+  <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-0.1.js"></script>
+
+  <!-- ... and the `amp-consent` extension. -->
+  <script async custom-element="amp-consent" src="https://cdn.ampproject.org/v0/amp-consent-0.1.js"></script>
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <style amp-custom>
+    * {
+      box-sizing: border-box;
+    }
+    amp-story * {
+      font-family: 'Helvitica Nueve', sans-serif;
+      color: white;
+    }
+    amp-story-page {
+      background: white;
+    }
+    amp-story h1 {
+      font-size: 46px;
+    }
+    amp-story h2 {
+      font-size: 36px;
+    }
+    amp-story p {
+      font-size: 16px;
+      line-height: 24px;
+    }
+    .bold {
+      font-weight: bold;
+    }
+    .bottom {
+      align-content: end;
+    }
+    .medium {
+      font-weight: 600;
+    }
+    .last {
+      padding-bottom: 65px;
+    }
+    .center {
+      text-align: center;
+    }
+    .icon {
+      background-image: url(/img/AMP-Brand-White-Icon.svg);
+      background-repeat: no-repeat;
+      background-size: 50px 50px;
+      height: 50px;
+      object-fit: contain;
+      width: 50px;
+    }
+    .byline {
+      letter-spacing: 1.28px;
+      padding-bottom: 58px;
+    }
+  </style>
+</head>
+  <body>
+
+    <!-- ## Implementing User Consent inside an AMP Story 
+      
+   The `amp-consent` will display the logo specified by `publisher-logo-src` in the consent dialog.
+    -->
+    <amp-story standalone bookend-config-src="<%host%>/json/bookend.json"
+               title="Story With User Consent"
+               publisher="Google"
+               poster-portrait-src="<%host%>/img/story_video_dog_cover.jpg"
+               publisher-logo-src="<%host%>/img/amp_logo_pink.svg">
+
+      <!--
+      The `amp-consent` component allows to specify the required user consent for this page.  A CORS endpoint can be specified via the `checkConsentHref` attribute. This endpoint will be queried via a POST request to determine if the consent UI specified for it needs to be shown. The response should look like this:
```

Comment:
we are explaining here what amp-consent does but above we are already mentioning it. Maybe move the explanation above?

Reply:
done